### PR TITLE
Add new citations for recapitulated results

### LINKED ISSUE #365: Revision: Indicate novel vs recapitulated findings
Throughout the manuscript when findings are reported, we should clarify whether they are a) new and/or to our knowledge not previously observed, or b) previously observed and/or in-line with expectations from previous research.

### PR DESCRIPTION
**Pull request by sjspielman**
Closes #365

This PR adds a couple new citations to round out this reviewer comment. Feedback on phrasing (especially the last sentence I modified -  is it too wordy?) is welcome!

### CODE REVIEW

File: content/03.results.md [hunk 1]
```diff
@@ -143,14 +143,14 @@ We detected canonical _SMARCB1/SMARCA4_ deletions or inactivating mutations in a
 
 #### High-grade gliomas
 <!--SAMPLECOUNT-->
-Across HGGs, we found that _TP53_ (57%, 35/61) and _H3F3A_ (52%, 32/61) were both most mutated and co-occurring genes (**Figure {@fig:Fig2}A and C**), followed by frequent mutations in _ATRX_ (30%, 18/61).
+Across HGGs, we found that _TP53_ (57%, 35/61) and _H3F3A_ (52%, 32/61) were both most mutated and co-occurring genes (**Figure {@fig:Fig2}A and C**), followed by frequent mutations in _ATRX_ (30%, 18/61) which is commonly mutated in gliomas [@doi:10.1080/14728222.2018.1487953].
```
Comment (jharenza) [inline content/03.results.md:146]:
```suggestion
Across HGGs, we found that _TP53_ (57%, 35/61) and _H3F3A_ (52%, 32/61) were both most mutated and co-occurring genes (**Figure {@fig:Fig2}A and C**), followed by frequent mutations in _ATRX_ (30%, 18/61), a gene commonly mutated in gliomas [@doi:10.1080/14728222.2018.1487953].
```

File: content/03.results.md [hunk 2]
```diff
@@ -166,7 +166,7 @@ We analyzed mutational co-occurrence across the OpenPBTA, using a single tumor f
...
 As expected, _TP53_ was the most frequently mutated gene across the OpenPBTA (8.4%, 56/666), significantly co-occurring with _H3F3A_ (OR = 32, 95% CI: 15.3 - 66.7, q = 8.46e-17), _ATRX_ (OR = 20, 95% CI: 8.4 - 47.7, q = 4.43e-8), _NF1_ (OR = 8.62, 95% CI: 3.7 - 20.2, q = 5.45e-5), and _EGFR_ (OR = 18.2, 95% CI: 5 - 66.5, q = 1.6e-4), with all of these driven by HGGs and consistent with previous reports [@doi:10.1016/j.ccell.2017.08.017; @doi:10.1093/neuonc/noaa251; @doi:10.1038/ng.2938].
 
-At the broad histology level, mutations in _CTNNB1_ significantly co-occurred with mutations in _TP53_ (OR = 42.9, 95% CI: 7 - 261.4, q = 1.63e-3) and _DDX3X_ (OR = 21.1, 95% CI: 4.6 - 96.3, q = 4.46e-3) in medulloblastoma.
+In medulloblastoma tumors, mutations in _CTNNB1_ significantly co-occurred with mutations in _TP53_ (OR = 42.9, 95% CI: 7 - 261.4, q = 1.63e-3), as has been previously observed [@doi:10.1200/JCO.2010.31.1670], as well as in _DDX3X_ (OR = 21.1, 95% CI: 4.6 - 96.3, q = 4.46e-3), which has also been previously identified as a commonly mutated gene in medulloblastomas [@doi:10.1038/nrc3410].
```
Comment (jharenza) [inline content/03.results.md:169]:
```suggestion
In medulloblastoma tumors, mutations in _CTNNB1_ significantly co-occurred with mutations in _TP53_ (OR = 42.9, 95% CI: 7 - 261.4, q = 1.63e-3), as previously reported [@doi:10.1200/JCO.2010.31.1670], as well as with _DDX3X_ (OR = 21.1, 95% CI: 4.6 - 96.3, q = 4.46e-3), another commonly mutated gene in medulloblastomas [@doi:10.1038/nrc3410].
```

Comment (jharenza) [reply content/03.results.md:169]: how about this?

**Review by jharenza — APPROVED** [review]
made some suggestions to shorten!

**Comment by sjspielman** [comment]
Ugh, sorry, I accidentally clicked `merge` in a post-lunch stupor!! I'm going to fix all this to grab your suggestions in a new PR!
